diff --git a/node_modules/country-flag-icons/1x1/SY.svg b/node_modules/country-flag-icons/1x1/SY.svg
index a3f6af1..9e09541 100644
--- a/node_modules/country-flag-icons/1x1/SY.svg
+++ b/node_modules/country-flag-icons/1x1/SY.svg
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="150 0 600 600"><path d="M0 0h900v600H0z" fill="#0f0f0f"/><path d="M0 0h900v400H0z" fill="#fff"/><path d="M0 0h900v200H0z" fill="#047a3f"/><path d="m176.26 375 48.738-150 48.738 150-127.6-92.705h157.72M626.256 375l48.738-150 48.738 150-127.6-92.705h157.72M401.252 375l48.738-150 48.738 150-127.6-92.705h157.72" fill="#ce1126"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="150 0 600 600"><path fill="#000001" d="M0 0h900v600H0z"/><path fill="#fff" d="M0 0h900v400H0z"/><path fill="#ce1126" d="M0 0h900v200H0z"/><path fill="#007a3d" d="m251 375 48.75-150 48.75 150-127.63-92.72h157.76M551 375l48.75-150 48.75 150-127.63-92.72h157.76"/></svg>
diff --git a/node_modules/country-flag-icons/3x2/SY.svg b/node_modules/country-flag-icons/3x2/SY.svg
index bb439bf..d893707 100644
--- a/node_modules/country-flag-icons/3x2/SY.svg
+++ b/node_modules/country-flag-icons/3x2/SY.svg
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 900 600"><path d="M0 0h900v600H0z" fill="#0f0f0f"/><path d="M0 0h900v400H0z" fill="#fff"/><path d="M0 0h900v200H0z" fill="#047a3f"/><path d="m176.26 375 48.738-150 48.738 150-127.6-92.705h157.72M626.256 375l48.738-150 48.738 150-127.6-92.705h157.72M401.252 375l48.738-150 48.738 150-127.6-92.705h157.72" fill="#ce1126"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 900 600"><path fill="#000001" d="M0 0h900v600H0z"/><path fill="#fff" d="M0 0h900v400H0z"/><path fill="#ce1126" d="M0 0h900v200H0z"/><path fill="#007a3d" d="m251 375 48.75-150 48.75 150-127.63-92.72h157.76M551 375l48.75-150 48.75 150-127.63-92.72h157.76"/></svg>
diff --git a/node_modules/country-flag-icons/string/1x1/SY.js b/node_modules/country-flag-icons/string/1x1/SY.js
index a8897b7..fbc00cb 100644
--- a/node_modules/country-flag-icons/string/1x1/SY.js
+++ b/node_modules/country-flag-icons/string/1x1/SY.js
@@ -1 +1 @@
-export default '<svg xmlns="http://www.w3.org/2000/svg" viewBox="150 0 600 600"><path d="M0 0h900v600H0z" fill="#0f0f0f"/><path d="M0 0h900v400H0z" fill="#fff"/><path d="M0 0h900v200H0z" fill="#047a3f"/><path d="m176.26 375 48.738-150 48.738 150-127.6-92.705h157.72M626.256 375l48.738-150 48.738 150-127.6-92.705h157.72M401.252 375l48.738-150 48.738 150-127.6-92.705h157.72" fill="#ce1126"/></svg>'
\ No newline at end of file
+export default '<svg xmlns="http://www.w3.org/2000/svg" viewBox="150 0 600 600"><path fill="#000001" d="M0 0h900v600H0z"/><path fill="#fff" d="M0 0h900v400H0z"/><path fill="#ce1126" d="M0 0h900v200H0z"/><path fill="#007a3d" d="m251 375 48.75-150 48.75 150-127.63-92.72h157.76M551 375l48.75-150 48.75 150-127.63-92.72h157.76"/></svg>'
diff --git a/node_modules/country-flag-icons/string/3x2/SY.js b/node_modules/country-flag-icons/string/3x2/SY.js
index 25157c7..29b3917 100644
--- a/node_modules/country-flag-icons/string/3x2/SY.js
+++ b/node_modules/country-flag-icons/string/3x2/SY.js
@@ -1 +1 @@
-export default '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 900 600"><path d="M0 0h900v600H0z" fill="#0f0f0f"/><path d="M0 0h900v400H0z" fill="#fff"/><path d="M0 0h900v200H0z" fill="#047a3f"/><path d="m176.26 375 48.738-150 48.738 150-127.6-92.705h157.72M626.256 375l48.738-150 48.738 150-127.6-92.705h157.72M401.252 375l48.738-150 48.738 150-127.6-92.705h157.72" fill="#ce1126"/></svg>'
\ No newline at end of file
+export default '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 900 600"><path fill="#000001" d="M0 0h900v600H0z"/><path fill="#fff" d="M0 0h900v400H0z"/><path fill="#ce1126" d="M0 0h900v200H0z"/><path fill="#007a3d" d="m251 375 48.75-150 48.75 150-127.63-92.72h157.76M551 375l48.75-150 48.75 150-127.63-92.72h157.76"/></svg>'
